### PR TITLE
travis - sudo: false, disable compiler

### PR DESCRIPTION
**Pull request by mattkime**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,17 +14,18 @@ branches:  # exclude tags
...
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
+# addons:
+#   apt:
+#    sources:
+#      - ubuntu-toolchain-r-test
+#    packages:
+#      - g++-4.8
```
Comment (mmcgahan) [inline .travis.yml]: might as well delete this since we can always revert later.

**Review by mmcgahan — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mattkime) [reply .travis.yml]: reverting code to add functionality would be very non-obvious to people who aren't you or me.

Comment (mmcgahan) [reply .travis.yml]: yeah - I think if it was needed later on someone would have to read the travis docs (or reference a repo that is already building sass-tools) and find out what to do, but we shouldn't keep around commented code that doesn't serve a purpose in the current system - it's just noise.

Comment (mattkime) [reply .travis.yml]: lets at least find a way to tell people how to enable this functionality if it becomes needed in the future.

Comment (mmcgahan) [reply .travis.yml]: Travis has our back - this is part of the default output for Node builds that don't specify a compiler:

screen shot 2017-04-13 at 11 53 24 am

That link [has all the info anyone would need](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements)

**Review by mmcgahan — APPROVED** [review]

**Pull request merged**